src/main/java/com/yanolja_final/domain/notice/controller/NoticeController.java

```diff
@@ -30,4 +33,13 @@ public ResponseEntity<ResponseDTO<RegisterNoticeResponse>> createNotice(
 
         return ResponseEntity.status(HttpStatus.valueOf(response.getCode())).body(response);
     }
+
+    @GetMapping
+    public ResponseEntity<ResponseDTO<List<NoticeListResponse>>> getNoticeList() {
+
+        ResponseDTO<List<NoticeListResponse>> response = noticeFacade.getNoticeList();
+
+        return ResponseEntity.status(HttpStatus.valueOf(response.getCode())).body(response);
+
```

Comment:
```suggestion
        ResponseDTO<List<NoticeListResponse>> response = noticeFacade.getNoticeList();
        return ResponseEntity.status(HttpStatus.valueOf(response.getCode())).body(response);
```

불필요한 줄바꿈 없도록 신경써주시면 좋을 것 같습니다!